packages/color-scale-block/package.json

```
@@ -0,0 +1,41 @@
+{
+    "name": "color-scale-block",
+    "private": true,
+    "author": "Frontify Developers <developers@frontify.com>",
+    "version": "1.0.0",
+    "main": "dist/index.js",
+    "scripts": {
+        "serve": "frontify-cli block serve",
+        "deploy": "frontify-cli block deploy",
+        "lint": "eslint src",
+        "lint:fix": "eslint --fix src",
+        "prettier": "prettier --check src",
+        "prettier:fix": "prettier --write src",
+        "typecheck": "tsc --noEmit"
+    },
+    "devDependencies": {
+        "@babel/core": "7.19.1",
+        "@frontify/eslint-config-react": "0.15.2",
+        "@frontify/frontify-cli": "5.1.2",
+        "@types/react": "17.0.50",
+        "@types/react-dom": "17.0.17",
+        "autoprefixer": "10.4.11",
+        "cypress": "10.8.0",
+        "eslint": "8.23.1",
+        "postcss": "8.4.16",
+        "prettier": "2.7.1",
+        "tailwindcss": "3.1.8",
+        "typescript": "4.8.3"
+    },
+    "dependencies": {
+        "@frontify/app-bridge": "3.0.0-beta.16",
+        "@frontify/fondue": "11.3.0",
```

```suggestion
        "@frontify/fondue": "12.0.0-beta.2",
```
As it has some breaking changes on `Button`